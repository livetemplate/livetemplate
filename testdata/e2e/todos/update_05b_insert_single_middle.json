{
  "10": "2023-01-01 11:15:00",
  "2": "10",
  "5": "4",
  "7": "3",
  "8": "25%",
  "9": [
    [
      "u",
      "todo-1",
      {
        "3": "#2"
      }
    ],
    [
      "u",
      "todo-3",
      {
        "3": "#3"
      }
    ],
    [
      "i",
      "todo-4",
      "after",
      {
        "0": "todo-5",
        "1": "",
        "2": "todo-5",
        "3": "#1",
        "4": "Configure CI/CD pipeline",
        "5": "â—‹",
        "6": "(Priority: Medium)"
      }
    ]
  ]
}
