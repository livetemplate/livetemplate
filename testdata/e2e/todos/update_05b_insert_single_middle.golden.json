{
  "1": "10",
  "4": "4",
  "6": "3",
  "7": "25%",
  "8": {
    "0": {
      "d": [
        {
          "0": "",
          "1": "todo-4",
          "2": "",
          "3": "Setup development environment",
          "4": "○",
          "5": " (Priority: High)"
        },
        {
          "0": "",
          "1": "todo-5",
          "2": "",
          "3": "Configure CI/CD pipeline",
          "4": "○",
          "5": " (Priority: Medium)"
        },
        {
          "0": "completed",
          "1": "todo-1",
          "2": "",
          "3": "Learn Go templates",
          "4": "✓",
          "5": " (Priority: High)"
        },
        {
          "0": "",
          "1": "todo-3",
          "2": "",
          "3": "Write documentation",
          "4": "○",
          "5": " (Priority: Low)"
        }
      ],
      "s": [
        "\n                    \u003cdiv class=\"todo-item ",
        "\" data-key=\"",
        "\"\u003e\n                        \u003cstrong\u003e",
        ":\u003c/strong\u003e\n                        ",
        "\n                        ",
        "\n                        ",
        "\n                    \u003c/div\u003e\n                    "
      ]
    },
    "s": [
      "\n                \u003cdiv class=\"todo-list\"\u003e\n                    ",
      "\n                \u003c/div\u003e\n                "
    ]
  },
  "9": "2023-01-01 11:15:00"
}